R/optionalInput.R

```diff
@@ -446,7 +446,7 @@ optionalSliderInputValMinMax <- function(inputId, label, value_min_max, label_he
 
   x <- value_min_max
 
-  if (!is.numeric(x)) stop("value_min_max is expected to be numeric")
+  checkmate::assert_numeric(x, .var.name = "value_min_max")
```

Comment:
Assert on argument and then assign to `x`, please. 
You can also extend to test for length (1 or 3) and values (middle must be the lowest).